bin/mail/2016-11-22-hoc-moana/send-real-emails

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# 490681 us-ca-uk teachers
+../send-to-mailing-list --limit 500000 2016-11-22-hoc-intl-moana ./teachers-us-ca-uk.csv
+
+# 274413 international teachers
+../send-to-mailing-list --limit 280000 2016-11-22-hoc-US-CA-UK-moana ./teachers-intl.csv
```

Comment:
I just noticed, these are mixed up. Obviously the intl should go to the intl list, and vice versa. Update forthcoming...